CHANGELOG.md

```diff
@@ -1,14 +1,17 @@
-# 0.0.116rc1 - Jul 14, 2023 - "Anchoring the Roadmap"
+# 0.0.116 - XXX, 2023 - "Anchoring the Roadmap"
 
 ## API Updates
 
  * Support for zero-HTLC-fee anchor output channels has been added and is now
    considered beta (#2367). Users who set
    `ChannelHandshakeConfig::negotiate_anchors_zero_fee_htlc_tx` should be
    prepared to handle the new `Event::BumpTransaction`, e.g. via the
-   `BumpTransactionEventHandler` (#2089). Users who set the same and wish to
-   accept inbound anchor-based channels must do so manually by setting
-   `UserConfig::manually_accept_inbound_channels` (#2368).
+   `BumpTransactionEventHandler` (#2089). Note that in order to do so you must
+   ensure you always have a reserve of available unspent on-chain funds to use
+   for CPFP. LDK currently makes no attempt to ensure this for you.
```

Comment:
so who is going to use anchor outputs with 0.0.116, `ldk-node` ? i’ve looked around `negotiate_anchors_zero_fee_htlc_tx` and our `CoinSelectionSource`, we don’t really have documentation about how to manage fee-bumping reserves, what do you think if I write some simple, conservative and over-collateralized docs for the lightningdevkit website, or do we prefer security-related doc to be under the git tree ?

Reply:
Happy to review the post for the LDK blog introducing anchor outputs when it’s ready (don’t hesitate to tag me on it). I’ll go to write more documentation, and I think it’s more appropriate to have them on the website (even if I know long-term it’s more maintenance work), as the very least we start to have a multitude of LDK deployment (mobile / large nodes / standard wallet) and generic documentation on fee-bumping reserves might be better suited here.

@ConorOkus Happy to have your thinking here, as you might have more visibility on downstream users, and if operational and safety documentation should go on the website or in struct/config docs, what is the best way to convey important information to the LDK users.

Reply:
I think I'll go for a `doc/fee-bumping-reserve.md` a la Linux and it can be copy-paste on the website if we wanna more docs there. Please go to land 0.0.116, I'll add them a posteriori during early 0.0.117 cycle.

Reply:
Here you go https://github.com/lightningdevkit/rust-lightning/pull/2450